app/models/split.rb

```diff
@@ -198,7 +217,7 @@ def registry_weights_must_be_integers
   end
 
   def name_not_underscored?
-    name && !underscored?(name)
+    name && !underscored?(name.split('.').last)
```

Comment:
Do we validate that the app name is snake cased too?

Reply:
(not seeing it on `app.rb`)

Reply:
Actually I wanted that more flexible so app name can match any repo name which often has dashes, depending on your platform persuasion. That relaxation is actually why I made this change.